site/documentation/content/api/command-router/index.md

```diff
@@ -46,6 +48,7 @@ The following table provides an overview of the properties a client needs to set
 | *subject*             | yes       | *properties*             | *string*  | MUST be set to `register-cmd-consumer`. |
 | *adapter_instance_id* | yes       | *application-properties* | *string*  | The identifier of the protocol adapter instance that currently handles commands for the device or gateway identified by the *device_id* property. |
 | *device_id*           | yes       | *application-properties* | *string*  | MUST contain the ID of the device that is subject to the operation. |
+| *send_event*          | no        | *application-properties* | *boolean* | If set to `true` [Time until Disconnect Notification]({{< relref "/concepts/device-notifications#time-until-disconnect-notification" >}}) with *ttd* value `-1` should be sent indicating the device is ready to receive commands. |
```

Comment:
Putting an "a" in front of "Time until Disconnect Notification" (also in other places) and a comma after `If set to true` (also in other table) would make this more readable in my view.

Reply:
agree, corrected.